### PR TITLE
minor change in language (typos, capitalization, etc...)

### PR DESCRIPTION
**Pull request by jaymanalastas**
minor change in language (typos, capitalization, etc...)

### CODE REVIEW

File: tutorials/beginner/decoupling_logic_and_execution.ipynb [hunk 1]
```diff
@@ -6,25 +6,25 @@
    "source": [
```
Comment (kvnkho) [inline tutorials/beginner/decoupling_logic_and_execution.ipynb:6]: <p>There seems to be an extra space here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (kvnkho) [inline tutorials/beginner/decoupling_logic_and_execution.ipynb:6]: <p>FugueWorkflow I think should not have the (). Either way we should make the two appearances here consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: tutorials/beginner/introduction.ipynb [hunk 2]
```diff
@@ -6,11 +6,11 @@
    "source": [
```
Comment (kvnkho) [inline tutorials/beginner/introduction.ipynb:6]: <p>Seems to be an extra base in the first two words</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: tutorials/beginner/decoupling_logic_and_execution.ipynb (hunk 1 above)
Comment (jaymanalastas) [reply tutorials/beginner/decoupling_logic_and_execution.ipynb:6]: done

File: tutorials/beginner/introduction.ipynb (hunk 2 above)
Comment (jaymanalastas) [reply tutorials/beginner/introduction.ipynb:6]: I'm not sure what you mean.

**Review by kvnkho — APPROVED** [review]

**Pull request merged**